src/kontrol/kdist/cheatcodes.md

```diff
@@ -633,11 +633,12 @@ This is needed in order to prevent overwriting the caller for subcalls.
          <prank>
            <active> true </active>
            <newCaller> NCL </newCaller>
-           <depth> CD </depth>
+           <depth> D </depth>
```

Comment:
There are a few other places where we duplicate variable names. Perhaps we should update those as well?